go/consensus/tendermint/seed.go

```diff
@@ -173,7 +174,10 @@ func populateAddrBookFromGenesis(addrBook p2p.AddrBook, doc *genesis.Document, o
 		var tmvAddr *p2p.NetAddress
 		tmvAddr, err := api.NodeToP2PAddr(&openedNode)
 		if err != nil {
-			return errors.Wrap(err, "tendermint/seed: failed to reformat genesis validator address")
+			logging.GetLogger("seed").Error("tendermint/seed: failed to reformat genesis validator address",
```

Comment:
Create a (global?) logger instance and use it here and below instead of creating a new instance each time.

Reply:
ya good idea